CHANGELOG.md

```diff
@@ -5,6 +5,45 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.7]
+
+### Added
+
+- Secrets are now able to be pushed up during a deploy in the added Secrets
+  view. Secret names can be added to the Configuration, and values supplied
+  individually in the Secrets view. (#2260, #2304, #2305)
+- Newly made Deployments can now be associated with previously deployed content
+  to easily update content on a Connect server. (#2285)
+- Views that require action now show a warning icon when collapsed to assist in
+  discovery (#2245)
+- Credentials are now persisted on OSes without keychains by storing them in the
+  user's home directory under `.connect-credentials`. If using a previous
+  version of the extension with a `.connect-credentials` dotfile remove it when
+  upgrading. (#1831)
```

Comment:
Is it worth referencing the linux package requirement here (`iNotifyTools`)?

Reply:
If I'm understanding correctly `inotify-tools` is a requirement for VS Code file watchers, and not our `.connect-credentials` feature. Is that correct? 

If so I think it can be mentioned in our install docs, but isn't actually a change for this release.